### ISSUE TITLE
Feature 'saveTagsAlphabetically' has been removed

### ISSUE THREAD

**Issue by vienai8d** [BoostIO/Boostnote#2699]
Feature 'saveTagsAlphabetically' can be controlled in preferences UI of Boostnote, but it looks like its core logic has been removed from master branch. 

<img width="317" alt="savetagsalphabetically in preferences ui" src="https://user-images.githubusercontent.com/31370233/49904318-22b6c200-fead-11e8-98b0-6eecb24ef796.png">

Specifically, the feature had been added by https://github.com/BoostIO/Boostnote/commit/8b4a9dd3259b6935e9fc2c6b346ca62d0ee02cdb#diff-c9e6aa56696b0ebb91171562f0891a52R90, but its core logic has been removed by https://github.com/BoostIO/Boostnote/commit/4e80e1dd03bcb1f94db81a563c57a48b1eea89b9#diff-c9e6aa56696b0ebb91171562f0891a52L90.

Is this correct?

**Comment by ZeroX-DG** [comment]
I still can use it in the current master.
![cas](https://user-images.githubusercontent.com/12984316/50040084-3f9af300-006f-11e9-9bd9-d7dbe7100443.gif)

**Comment by vienai8d** [comment]
Probably, it is expected that tags of a note are saved in order of its name alphabetically when 'saveTagsAlphabetically' is enable.
But, in the current master, the tags are saved in order of the creation as below (In this example, I turned off 'showTagsAlphabetically').
![savetagsalphabetically_off](https://user-images.githubusercontent.com/31370233/50041055-d24c8980-0091-11e9-9050-d3ede938b2eb.gif)

I fixed this feature in reference to 
https://github.com/BoostIO/Boostnote/commit/8b4a9dd3259b6935e9fc2c6b346ca62d0ee02cdb#diff-c9e6aa56696b0ebb91171562f0891a52R90 and saved as [saveTagsAlphabetically ](https://github.com/vienai8d/Boostnote/tree/saveTagsAlphabetically) branch.
In my fixed version, the tags the tags are saved in order of its name alphabetically.
![savetagsalphabetically_onf](https://user-images.githubusercontent.com/31370233/50041232-01182f00-0095-11e9-8336-60e059e0a0f0.gif)
